pkg/assetvalidator/errors.go

```diff
@@ -18,3 +18,21 @@ func (mve MultiValidatorError) Error() string {
 	}
 	return strings.Join(errstrings, "\n")
 }
+
+// ValidationFailure occurs when the process of validation fails
+// unexpectantly, and not due to an invalid attributed asset
+// Examples of usage would include a bad http call
+type ValidationFailure struct {
```

Comment:
Suggestion: I think the error types should be a little more clear and consistent here.

How about:

- Individual validation failures return a ValidationFailure error type that is just an error implementation with no additional state.
- Individual validation errors return a similar ValidationError error type.
- The MultiValidator returns either a MultiValidationFailure or MultiValidationError error type
- Since the handler will always use a MultiValidator, regardless of how many individual validators are configured, the handler can then only check for the MultiValidationFailure and MultiValidationError types.

What do you think?